sound/soc/intel/boards/sof_sdw.c

```diff
@@ -870,6 +870,31 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		},
 		.dai_num = 1,
 	},
+	{
```

Comment:
the commit message is a bit misleading.

Suggested edit:

"
RT722 is a multi-function codec which supports headset,
amp, and dmic functions. Each function provides a DAI which can be used in different dailinks.

The RT711 supports up to 3 SoundWire lanes, but that should not have any impact in the machine driver: the lanes are allocated and controlled by the manager and bandwidth allocation algorithm.
"